### ISSUE TITLE
Accessibility: `<aside>` cannot be opened with keyboard controls and screen reader

### ISSUE THREAD

**Issue by virginiaMae** [Doenet/DoenetML#616]
When navigating using keyboard controls and a screen reader, a `<hint>` can be entered and opened by clicking "Return". However, this does not work with an `<aside>`, the cursor jumps back out to the main article upon hitting "Return". 
See https://beta.doenet.org/activityViewer/vnYfvxJqrmtPHuNQmTFZ2B for an example
